*** A casual runner records the average time it takes him to sprint 100 meters. He completes the dash 34 times under identical conditions and finds that the mean of these is 14.22 seconds. Assume that he knows the standard deviation of his runs is σX = 2.9 seconds.***

*** a. Construct and interpret a 90 percent confidence interval for the true mean time. ***

In [3]:
time.mean <- 14.22
time.sd <- 2.9
n <- 34
time.se <- time.sd/sqrt(n)
time.mean + c(-1,1)*qnorm(p=0.9)*time.se

[1] 13.58263 14.85737

We are 90% confident that the true mean of the time it takes the runner to sprint 100 meters is somewhere between 13.582 and 14.857 seconds. 

*** b. Repeat (a), but this time, assume that the standard deviation is not known and that s = 2.9 is estimated from the sample. How, if at all, does this change the interval? ***

In [4]:
time.mean <- 14.22
time.sd <- 2.9
n <- 34
time.se <- time.sd/sqrt(n)
time.mean + c(-1,1)*qt(p=0.9, df = n -1)*time.se

[1] 13.5696 14.8704

We are 90% confident that the true mean of the time it takes the runner to sprint 100 meters is somewhere between 13.569 and 14.870 seconds. 

#### *** In a particular country, the true proportion of citizens who are left handed or ambidextrous is unknown. A random sample of 400 people is taken, and each individual is asked to identify with one of three options: right-handed only, left-handed only, or ambidextrous. The results show that 37 selected left-handed and 11 selected ambidextrous ***

*** c. Calculate and interpret a 99 percent CI for the true proportion of left-handed-only citizens ***

In [6]:
p.hat <- .37
n <- 400
sd.error <- sqrt(p.hat*(1-p.hat)/n)
p.hat + c(-1,1)*qnorm(0.99)*sd.error

[1] 0.3138415 0.4261585

*** d. Calculate and interpret a 99 percent CI for the true proportion of citizens who are either left-handed or ambidextrous. ***

In [7]:
p.hat <- 0.37 + 0.11
n <- 400
p.se <- sqrt(p.hat*(1-p.hat)/n)
p.hat + c(-1,1)*qnorm(0.99)*p.se

[1] 0.4218878 0.5381122